src/stories/Checkbox.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from "react";
+
+interface ICheckbox {
+  /*
+  Whether the checkbox is checked.
+  */
+  isChecked?: boolean;
+  /*
+  The text to display next to the checkbox.
+  */
+  text?: string;
+}
+
+// Clicking the checkbox is not updating the initial isChecked value;
+// Can you update storybook controls by only interacting with the component?
+export const Checkbox = ({
+  isChecked = false,
+  text = "",
+  ...props
+}: ICheckbox) => {
+  const [checked, setChecked] = useState<boolean>(isChecked);
+  function handleChange(event: React.ChangeEvent<HTMLInputElement>) {
+    setChecked(event.target.checked);
+  }
+
+  return (
+    <div className="py-2">
+      <input
+        onChange={handleChange}
+        id="check"
+        type="checkbox"
+        className="hidden peer"
+        checked={checked || isChecked}
+        defaultChecked={isChecked}
```

Comment:
is this necessary?

Reply:
Probably not 😅 I think I was messing with the `checked` and `isChecked` values to try to get the `checked` Storybook control value to be updated when the checkbox element is interacted with. I'll take another look at this to clean it up!